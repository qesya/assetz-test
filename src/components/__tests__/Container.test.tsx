/* eslint-disable testing-library/prefer-screen-queries */
import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Container from '../Container';

const props = {
	label: 'name',
	value: '1',
	onChange: jest.fn(),
}

describe('<List />', () => {
	test('should match snapshot', () => {
		const { asFragment } = render(
			<Container {...props} />
		);
		expect(asFragment()).toMatchSnapshot();
	});
});
