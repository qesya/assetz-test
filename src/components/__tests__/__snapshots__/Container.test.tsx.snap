// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ goEvZo"
    label="name"
    value="1"
  />
</DocumentFragment>
`;
